ui/src/app/views/workspace/component.spec.ts

```diff
@@ -110,6 +111,17 @@ describe('WorkspaceComponent', () => {
         .toMatch('Log in to view workspace.');
     });
   }));
+
+  it('fetches the correct workspace', fakeAsync(() => {
+    workspacePage.fixture.componentRef.instance.ngOnInit();
+    updateAndTick(workspacePage.fixture);
+    updateAndTick(workspacePage.fixture);
+    expect(workspacePage.cdrText.nativeElement.innerText)
+      .toMatch('Fake CDR Version');
```

Comment:
These bare string seem a bit fragile. Can you pull these out into a common variable and use it in both places?